src/hooks.tsx

```diff
@@ -23,7 +23,7 @@ function useKeyPress (targetKey: string, onKeyDown: () => void, onKeyUp: () => v
 }
 
 const useAnimationFrame = (enabled: boolean, callback: (time: number, delta: number) => void, deps: React.DependencyList): void => {
-  const frame = useRef<number>()
+  const frame = useRef<number>(null)
```

Comment:
`eslint`/`tslint` was mad about it not having an initial value, so I gave it null to start with.